charts/bpdm-certificate-management/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on Keep a Changelog (https://keepachangelog.com/en/1.0.0/),
+
+## [0.0.1]  - 2023-11-24
```

Comment:
A `CHANGELOG.md` is not required by TRG, but totaly fine to have one for the Helm Chart if you want. But then the content should match the Helm chart: the helm chart version will be _0.0.1-alpha.1_, not _0.0.1_ after first run of helm chart releaser workflow.

Reply:
Removed content for 0.0.1 and targeting to keep only release version tags in future otherwise file will be bulky if we keep on adding alpha versions.